### DISCUSSION TITLE
When you place pylon it may not work

### DISCUSSION THREAD

**Discussion by grechnevi** [Pryaxis/TShock#1819] [Q&A]
Player places pylon, then this player don't see it. Other players can see it but can't interact and use it.
Same player can see this pylon after re-login but can't interact and use it.

If the player places pylon two times (so gold cost = x2) on the same spot, then this pylon does work.

**Comment by hakusaro** [comment]
This is a known issue -- we're working on it in: https://github.com/Pryaxis/TShock/issues/1795.
